recipes/koolstof/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "koolstof" %}
+{% set version = "0.26" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/koolstof-{{ version }}.tar.gz
+  sha256: 0fd636e0410844f6c8b6e70794f981a0a5bf92f01544f6f7fc0f79439103ce9c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - calkulate
```

Comment:
You probably don't need all the run dependencies on host. Unless this package imports itself at build time.

Reply:
Thanks - but this failed the automated tests when I didn't have them there - I'm not sure why...

Reply:
Probably because the pkg imports itself at build time.